tests/console/redis.pm

```diff
@@ -37,7 +38,7 @@ sub run {
     validate_script_output('redis-cli get foo', sub { m/bar/ });
     validate_script_output('redis-cli pfselftest', sub { m/OK/ });
     validate_script_output('redis-cli flushdb', sub { m/OK/ });
-    validate_script_output('redis-cli get foo', sub { m/(nil)/ });
+    validate_script_output('redis-cli get foo', sub { m/(nil)/ }) if (!is_svirt);
```

Comment:
This is not needed. The output is just not `nil` but `\n` so it can be done better with some perl magic :smile:

Reply:
Maybe this can help: https://perlmaven.com/check-if-string-is-empty-or-has-only-spaces-in-perl

Reply:
I still don't understand this, this is working on	qemu_x86_64. Do you mean to use sub { m/(nil) \n/ } ?
thanks!

Reply:
Something like that. In qemu backend it is `(null)` but in `svirt` it's probably `\n`